data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd

```diff
@@ -397,7 +405,7 @@ deer_trap_clean <- deer_rst %>%
          site = stream,
```

Comment:
This isn't working for me anymore either.

Reply:
updated